src/Annotator.js

```diff
@@ -99,7 +100,7 @@ class Annotator extends EventEmitter {
         }
 
         if (!this.container) {
-            this.emit(ANNOTATOR_EVENT.loadError, this.localized.loadError);
+            this.emit(ANNOTATOR_EVENT.error, this.intl.formatMessage(messages.annotationsLoadError));
```

Comment:
I left this comment before, but why did `loadError` change to `error` here? Was there a bug?

Reply:
Yes, there is no loadError key on the ANNOTATOR_EVENT constant so the errors were being called with undefined . There is another instance of it being called that I am going to push to the PR...